src/Compilers/CSharp/Test/Emit/Emit/CovariantReturnTests.cs

```diff
@@ -304,7 +313,74 @@ static void Main()
             var expectedOutput = @"";
             CompileAndVerify(compilation, expectedOutput: expectedOutput, verify: Verification.Skipped);
         }
-    }
+
+        [ConditionalFact(typeof(CovariantReturnRuntimeOnly))]
+        public void CheckPreserveBaseOverride_01()
```

Comment:
>CheckPreserveBaseOverride_01 [](start = 20, length = 28)

Would it make sense to have similar test with compilation references instead of image references?

Reply:
The compile-time behavior is already tested in the symbol tests. If you think it would be useful I can parameterize this test by the kind of emit used.

---
In reply to: [461799460](https://github.com/dotnet/roslyn/pull/46367#discussion_r461799460) [](ancestors = 461799460)

Reply:
>The runtime cannot run against compilation references.

Could you elaborate please? I am certain we can execute an verify output for a compilation which is utilizing compilation references.

---
In reply to: [461833574](https://github.com/dotnet/roslyn/pull/46367#discussion_r461833574) [](ancestors = 461833574,461799460)

Reply:
The runtime requires actual MSIL bytecodes to execute.  In any case, the compile-time behavior for these scenarios are tested elsewhere.

---
In reply to: [461945536](https://github.com/dotnet/roslyn/pull/46367#discussion_r461945536) [](ancestors = 461945536,461833574,461799460)